_saas-integrations/google-sheets/v1/google-sheets-v1.md

```diff
@@ -0,0 +1,134 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Google Sheets
+permalink: /integrations/saas/google-sheets
+keywords: google-sheeets, integration, schema, etl google-sheeets, google-sheeets etl, google-sheeets schema
+layout: singer
+# input: false
+
+key: "google-sheeets-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "google-sheeets"
```

Comment:
There's an extra `e` in `sheets` - this is why none of the schemas are showing up.